### PR TITLE
Tools: Testbench: Add useful script for run and profile

### PR DESCRIPTION
**Pull request by singalsu**

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]

File: tools/testbench/README.md [hunk 1]
```diff
@@ -105,50 +172,83 @@ octave -q --eval "pkg load signal io; [n_fail]=process_test('eq-iir', 32, 32, 48
...
+   -t tools/build_tools/test/topology/test-playback-ssp5-mclk-0-I2S-drc-s32le-s32le-48k-24576k-codec.tplg \
+   -i in.raw -o out.raw 2> trace.txt
+```
+
+Then convert the profiler data to readable format and check it.
 
-The debugging and profiling of IPC4 testench is similar as with IPC3.
+Note: Current version of testbench does not have functional quiet mode
+"-q" switch to suppress trace. Due all debug traces print
+out. Majority of performance is spent in printing.
```
Comment (kv2019i) [inline tools/testbench/README.md:249]: "Due all debug traces print out." is a bit unclear, what do you mean by this? Also "Majority of performance" sounds a bti off, "Majority of cycles" ?

**Review by kv2019i — COMMENTED** [review]
Excellent stuff @singalsu ! One minor note on documentation for the second patch, please see inline.

File: tools/testbench/README.md (hunk 1 above)
Comment (singalsu) [reply tools/testbench/README.md:249]: Thanks, oops, what did I write. Also the -q option was removed as duplicate of -d , need to fix this.

Comment (singalsu) [reply tools/testbench/README.md]: Fixed now!

**Review by kv2019i — APPROVED** [review]

**Pull request merged**